votes/vote-20230629-sponsorship-policy.txt

```diff
@@ -0,0 +1,13 @@
+Topic:As per the recommendation of the Working Group, the OMC to accept the Sponsorship Policy as of commit 89a7619dc55a9a59462b4e464c5f44a436e4b3e1.
```

Comment:
The date is definitely incorrect here.  Policy votes have a mandated two week minimum waiting period.  I.e. starting the vote on 19th Oct 2023 is the bare minimum here & likely later.  Moreover, votes cannot be _called_ before the two week period has expired, so that there is time to comment & review.  I.e. this needs to be called after two weeks.

That is unless the OMC is willing to consider voting in an exception.  That would either need to be raised as a separate OMC vote or on an OMC call.